WalletWasabi.Fluent/Helpers/LinuxStartupHelper.cs

```diff
@@ -17,14 +17,16 @@ public static async Task AddOrRemoveDesktopFileAsync(bool runOnSystemStartup)
 		{
 			string pathToExec = EnvironmentHelpers.GetExecutablePath();
 
+			string pathToExecWithArgs = string.Concat(pathToExec, " ", StartupHelper.SilentArgument);
```

Comment:
Sorry I was not clear. 
```suggestion
			string pathToExecWithArgs = $"{pathToExec}  {StartupHelper.SilentArgument}";
```